src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFSamplersTest.java

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.math3.random.RandomGenerator;
+import org.apache.commons.math3.random.RandomGeneratorFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by davidben on 11/25/15.
+ */
+public class AFSamplersTest {
+    private static final int RANDOM_SEED = 42;
+    private static final RandomGenerator rng = RandomGeneratorFactory.createRandomGenerator(new Random(RANDOM_SEED));
+    final int numSamples = 500;
+
+    final static double averageHetsPerSegment = 20;
+    final static int numSegments = 100;
+    final static int averageDepth = 50;
+
+    final static double meanBias = 1.1;
+    final static double biasRate = 100;
+    final static double pOutlier = 0.02;
+
+    // note: the following tolerances could actually be made much smaller if we used more segments and/or
+    // more hets -- most of the error is the sampling error of a finite simulated data set, not numerical error of MCMC
+    final double minorFractionTolerance = 0.02;
+    final double meanBiasTolerance = 0.02;
+    final double biasRateTolerance = 500;
+    final double pOutlierTolerance = 0.02;
+
+    final static AFState initialState;
+    final static AFData data;
+
+    static {
```

Comment:
the logic here doesn't seem complicated enough to necessitate this static initialization block (the use of which is idiosyncratic here, i think).